### PR TITLE
Adds missing translations of zone names

### PR DESCRIPTION
**Pull request by madsnedergaard**
## Description

Just noticed that https://api.electricitymap.org/v3/zones returns a bunch of unknown names because we're still missing English translations for new zones and aggregates. This should ideally not happen, as we're also leaving areas on the map with weird titles :(

Related to #4782 for the Indian zones - I just went ahead and used the names from this Wikipedia: https://en.wikipedia.org/wiki/National_Grid_(India)#:~:text=Individual%20State%20grids%20were%20interconnected,between%20States%20in%20each%20region.

### CODE REVIEW

File: web/public/locales/en.json [hunk 1]
```diff
@@ -775,6 +778,24 @@
     "IM": {
       "zoneName": "Isle of Man"
     },
+    "IN": {
+      "zoneName": "India"
+    },
+    "IN-EA": {
+      "zoneName": "Eastern"
```
Comment (VIKTORVAV99) [inline web/public/locales/en.json:785]: Should we use the style we use for Italy here or the one we use for Norway, Sweden, and Denmark?
I.e. should it be:
```suggestion
      "zoneName": "Eastern India"
```

**Review by VIKTORVAV99 — APPROVED** [review]
Just a thought about the naming, we should strive to have a unified naming schemes.

This should be considered non-blocking.

Related issue: [ELE-1574](https://linear.app/electricitymaps/issue/ELE-1574/consistently-name-all-subzones)

**Review by VIKTORVAV99 — CHANGES_REQUESTED** [review]
Oh I just noticed we miss the country name for the India Subzones.

File: web/public/locales/en.json (hunk 1 above)
Comment (madsnedergaard) [reply web/public/locales/en.json:785]: We're a bit inconsistent here, as Italy for example don't have it:
Screenshot 2023-01-10 at 22 17 50

I personally think it looks better like this (the "North Sweden (Sweden)" part is weird), but we then do need to make sure that the country is always included next to any usage - so I guess that option is at least safer and easier to use :)

**Review by VIKTORVAV99 — APPROVED** [review]
Let's get this merged then we can evaluate what format we should use for the subzone names separately.

**Comment by madsnedergaard** [comment]
Thanks Viktor, forgot to push the change :)

**Pull request merged**